docs/sources/open-source/_index.md

```diff
@@ -187,3 +187,12 @@ Grafana OnCall supports Twilio SMS and phone call notifications delivery. If you
 
 1. Set `GRAFANA_CLOUD_NOTIFICATIONS_ENABLED` as **False** to ensure the Grafana OSS <-> Cloud connector is disabled.
 1. From your **OnCall** environment, select **Env Variables** and configure all variables starting with `TWILIO_`. 
+
+## Email Setup
+Grafana OnCall is capable of sending emails using SMTP as a user notification step. To setup email notifications, populate the following env variables with your SMTP server credentials:
```

Comment:
may be write that step called "Notify by email" and which page is that

Reply:
Added in https://github.com/grafana/oncall/pull/621/commits/e85ddf69b04338ccc6015232e243fd5cea984f9d